RetailSuiteAccounting/Integration/06_ManageFixedBookings.md

```diff
@@ -0,0 +1,182 @@
+[!!User Interface fixed bookings](../UserInterface/02f_FixedBookings.md)  
+[!!User Interface select fiscal year](../UserInterface/00a_FiscalYear.md)  
+[!!Select a fiscal year](../Operation/01_SelectFiscalYear.md)  
+
+# Manage the fixed bookings
+
+The *FIXED BOOKINGS* tab in the *Accounting* module enables to automate recurring bookings. The fixed bookings to be automated can be applied when selecting the fiscal year, see [Select the fiscal year](../Operation/01_SelectFiscalYear.md). By doing so, the system will apply automatically the setup fixed bookings in the corresponding fiscal year and journal.
+
+To be able to create fixed bookings, an existing batch must be selected or a new one must be created first.
+
+
+## Create a fixed booking batch
+
+You can create a fixed booking batch that is not yet available in the system.
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](./01_RunAccountingWizard.md).
+- A valid fiscal year is created, see [Create a fiscal year](./04_ManageFiscalYear.md#create-a-fiscal-year).
+
+#### Procedure
+
+*Accounting > Select the fiscal year > Settings > Tab FIXED BOOKINGS*
+
+![Create a fixed booking batch](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/CreateFixedBookingBatch.png "[Create a fixed booking batch]")
+
+1. Click the *Fixed booking batch* drop-down list and select the *New batch* option.
+
+  > [Info] Per default, the drop-down list displays the *New batch* option.
+
+2. Enter a batch name in the *Name* field.  
+
+3. Click the [CREATE] button.   
+A new window opens where the new fixed bookings can be registered.
```

Comment:
```suggestion
The *Fixed bookings &ndash; Batch* window is displayed.
```
Bild einfügen

Nächster Schritt: For the next steps to create a fixed booking, follow the procedure [Create a fixed booking]/#create-fixed-booking).